azurelinuxagent/common/protocol/wire.py

```diff
@@ -796,9 +799,7 @@ def update_host_plugin():
                     return
 
                 def incarnation_changed():
-                    last_incarnation = None
-                    if os.path.isfile(incarnation_file):
-                        last_incarnation = fileutil.read_file(incarnation_file)
+                    last_incarnation = self.goal_state.incarnation
```

Comment:
sorry, i missed this one before. this should also come in the PR that removes the files

Reply:
Reverted